src/ServiceControl.Monitoring/MetricsHandler.cs

```diff
@@ -1,20 +1,20 @@
 ﻿namespace ServiceControl.Monitoring.ServiceControl.Monitoring
 {
     using System.Threading.Tasks;
-    using Metrics.Json;
     using NServiceBus;
+    using NServiceBus.Metrics;
 
-    public class MetricsHandler : IHandleMessages<JsonMetricsContext>
+    public class MetricsHandler : IHandleMessages<MetricReport>
     {
         static readonly Task<int> CompletedTask = Task.FromResult(0);
-        readonly PublisherConsumer<JsonMetricsContext> publisherConsumer;
+        readonly PublisherConsumer<MetricReport> publisherConsumer;
 
-        public MetricsHandler(PublisherConsumer<JsonMetricsContext> publisherConsumer)
+        public MetricsHandler(PublisherConsumer<MetricReport> publisherConsumer)
```

Comment:
Maybe out of scope of this PR but why going through `PublisherConsumer` instead of injecting `NServiceBusReceivedMetricContext` to the handler via container?